### PR TITLE
feat: Implement arrow up/down/left/right event handler. #708

### PR DESCRIPTION
**Pull request by LucasXu0**

### CODE REVIEW

File: frontend/app_flowy/packages/flowy_editor/example/lib/main.dart [hunk 1]
```diff
@@ -97,6 +97,21 @@ class _MyHomePageState extends State<MyHomePage> {
             return FlowyEditor(
               editorState: _editorState,
               keyEventHandler: const [],
+              shortCuts: [
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_editor/example/lib/main.dart:100]: shortcuts?

File: frontend/app_flowy/packages/flowy_editor/example/lib/plugin/selected_text_node_widget.dart [hunk 2]
```diff
@@ -108,6 +110,30 @@ class _SelectedTextNodeWidgetState extends State<_SelectedTextNodeWidget>
     return _renderParagraph.localToGlobal(offset);
   }
 
+  @override
+  Offset getLeftOfOffset() {
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_editor/example/lib/plugin/selected_text_node_widget.dart:114]: Backword or forward

Comment (LucasXu0) [reply frontend/app_flowy/packages/flowy_editor/example/lib/plugin/selected_text_node_widget.dart:114]: Sounds good.

File: frontend/app_flowy/packages/flowy_editor/lib/service/floating_shortcut_service.dart [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  @override
+  State<FloatingShortCut> createState() => _FloatingShortCutState();
+}
+
+class _FloatingShortCutState extends State<FloatingShortCut>
+    with FlowyFloatingShortCutService {
+  OverlayEntry? _floatintShortcutOverlay;
+
+  @override
+  void showInOffset(Offset offset, LayerLink layerLink) {
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_editor/lib/service/floating_shortcut_service.dart:34]: Missing documentation for showInOffset.

File: frontend/app_flowy/packages/flowy_editor/lib/service/service.dart [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import 'package:flowy_editor/service/floating_shortcut_service.dart';
+import 'package:flowy_editor/service/selection_service.dart';
+import 'package:flutter/material.dart';
+
+class FlowyService {
```
Comment (appflowy) [inline frontend/app_flowy/packages/flowy_editor/lib/service/service.dart:5]: _EditorGloableKeyDefine?

File: frontend/app_flowy/packages/flowy_editor/example/lib/main.dart (hunk 1 above)
Comment (LucasXu0) [reply frontend/app_flowy/packages/flowy_editor/example/lib/main.dart:100]: done

**Pull request merged**